addons/binding/org.openhab.binding.hdpowerview/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="hdpowerview"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <bridge-type id="hub">
+        <label>PowerView Hub</label>
+        <description>Hunter Douglas PowerView Hub</description>
+        
+        <config-description>
+            <parameter name="ipAddress" type="text" required="true">
+                <label>IP</label>
+                <description>The IP of the PowerView Hub.</description>
+            </parameter>
+            <parameter name="refresh" type="integer" required="false">
+                <label>Refresh Interval</label>
+                <description>The number of milliseconds between fetches of the PowerView Hub shade state.</description>
+                <default>60000</default>
+            </parameter>
+        </config-description>
+    </bridge-type>
+    
+    <thing-type id="shade">
+        <label>PowerView Shade</label>
+        <description>Hunter Douglas PowerView Shade</description>
+        <channels>
+            <channel id="position" typeId="shade-position"/>
+            <channel id="vane" typeId="shade-vane"/>
+            <channel id="batteryLow" typeId="shade-batteryLow"/>
```

Comment:
Please use the [existing system channel type](https://www.eclipse.org/smarthome/documentation/development/bindings/thing-definition.html#channels) `system.low-battery` instead.